libvast/src/system/exporter.cpp

```diff
@@ -108,6 +107,13 @@ behavior exporter(stateful_actor<exporter_state>* self, expression expr,
       self->quit(msg.reason);
     }
   );
+  self->set_down_handler(
+    [=](const down_msg& msg) {
+      VAST_DEBUG(self, "received DOWN from", msg.source);
+      if (msg.reason == exit_reason::user_shutdown)
+        shutdown(self);
```

Comment:
:+1:

Wanted to write: "[...] prompts the NODE **to** shutdown related ~the~ EXPORTERs with a down_msg?". What I mean is that the process that receives the CTRL + C might be a different process than the process that hosts the EXPORTER, right? And I wanted to make sure that the signal is still forwarded.